app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[destroy update]
+  # before_action :authenticate_user!
+
+  def index
+    @reservations = Reservation.includes(:doctor).all
+    render json: { reservations: @reservations }
+  end
+
+  def create
+    @reservation = Reservation.new(reservation_params)
+    # @reservation.user = current_user
```

Comment:
- [ ] Kindly, remove this old commented code in order to improve the repo's cleanliness. 👍

Reply:
Thank you @Meltrust , we will remove it.